sources/platform/console/two-factor-authentication.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Two-factor authentication
+description: Learn about Apify Console's two-factor authentication process and how to set it up.
+sidebar_position: 5
+category: platform
+slug: /console/two-factor-authentication
+---
+
+**Learn about Apify Console's account two-factor authentication process and how to set it up.**
+
+---
+
+If you use your email and password to sign in to Apify Console, you can enable two-factor authentication for your account. This will add an extra layer of security to your account and prevent anyone who gains access to your password from signing in to your account.
+
+## Setting up two-factor authentication
+
+To set up two-factor authentication, go to your [account settings](https://console.apify.com/account/settings) and scroll down to the **Two-factor authentication** section. Currently, there is only one option, which is the **Authenticator app**. If you have two-factor authentication already enabled, there will be a label **enabled** next to it.
+
+![Apify Console two-factor authentication section](./images/console-account-two-factor-disabled.png)
+
+If it's not enabled, click on the **Enable** button. You should see the two-factor setup view.
+
+### Authenticator app setup
+
+![Apify Console setup two-factor authentication - app](./images/console-two-factor-app-setup.png)
+
+In this view, you can use your favorite authenticator app to scan the QR code. We recommend using Google Authenticator ([Google play store](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US)/[Apple app store](https://apps.apple.com/us/app/google-authenticator/id388497605)) or [Authy](https://authy.com/) but any other authenticator app should work as well.
```

Comment:
We might consider adding google play / apple store links for Authy as well, just for consistency sake, something like, also Play Store and App Store should be capitalized.

```suggestion
In this view, you can use your favorite authenticator app to scan the QR code. We recommend using Google Authenticator ([Google Play Store](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US)/[Apple App Store](https://apps.apple.com/us/app/google-authenticator/id388497605)) or [Authy](https://authy.com/)([Google Play Store](https://play.google.com/store/apps/details?id=com.authy.authy)/[Apple App Store](https://itunes.apple.com/us/app/authy/id494168017) but any other authenticator app should work as well.
```